src/league.js

```diff
@@ -529,15 +528,15 @@ league_attributes = {
     'formatTeamCaptainResponse':function(teamName) {
         var self = this;
         return Q.fcall(function() {
-            if (self._teams.length == 0) {
+            if (self._teams.length === 0) {
                 return "The {name} league does not have captains".format({
                     name: self.options.name
                 });
             }
             teams = _.filter(self._teams, function(t) {
-                return t.name.toLowerCase() == teamName.toLowerCase()
+                return t.name.toLowerCase() === teamName.toLowerCase()
```

Comment:
This is the same case as below, we should use _.isEquals for both or neither. I suspect that toLowerCase() is pretty consistent about what types it returns, so === is probably fine.